Jenkinsfile

```diff
@@ -47,9 +47,15 @@ def job = {
             "master": "master"
     ]
 
+    stage("Check compilation compatibility with Scala 2.12") {
+        sh "./gradlew clean assemble spotlessScalaCheck checkstyleMain checkstyleTest spotbugsMain " +
+                 "--no-daemon --stacktrace -PxmlSpotBugsReport=true -PscalaVersion=2.12"
+        }
+
+
     stage("Compile and validate") {
         sh "./gradlew clean assemble spotlessScalaCheck checkstyleMain checkstyleTest spotbugsMain " +
-                "--no-daemon --stacktrace --continue -PxmlSpotBugsReport=true"
```

Comment:
Out of curiosity, was there a reason to remove this?

Reply:
This is a bug, we should not continue if compilation fails. We should continue if tests fail because we have the generated test results and those are then used to mark the build as unstable.